diff --git a/home/user/projects/openwrt-home/1.c b/home/user/projects/openwrt-home/openwrt-AC10/build_dir/target-mipsel_24kc_musl/libgpg-error-1.12/src/strerror-sym.c
index 02b47da623..ed1c80976f 100644
--- a/home/user/projects/openwrt-home/1.c
+++ b/home/user/projects/openwrt-home/openwrt-AC10/build_dir/target-mipsel_24kc_musl/libgpg-error-1.12/src/strerror-sym.c
@@ -42,15 +42,15 @@ gpg_strerror_sym (gpg_error_t err)
       int idx;
 
       code &= ~GPG_ERR_SYSTEM_ERROR;
-      idx = errnos_msgidxof (code);
+      idx = msgidxof (code);
       if (idx >= 0)
-	return errnos_msgstr + errnos_msgidx[idx];
+        return msgstr[idx];
       else
-	return NULL;
+        return NULL;
     }
 
   if (msgidxof (code) == msgidxof (GPG_ERR_CODE_DIM))
     return NULL;
 
   return msgstr + msgidx[msgidxof (code)];
-}
\ No newline at end of file
+}
